src/status_im/ui/screens/extensions/add/events.cljs

```diff
@@ -1,23 +1,32 @@
 (ns status-im.ui.screens.extensions.add.events
-  (:require [re-frame.core :as re-frame]
+  (:require [pluto.registry :as registry]
+            [re-frame.core :as re-frame]
             [status-im.extensions.core :as extensions]
             [status-im.ui.screens.navigation :as navigation]
             [status-im.utils.handlers :as handlers]
             [status-im.utils.fx :as fx]))
 
 (re-frame/reg-fx
- :extension/load
- (fn [[url follow-up-event]]
-   (extensions/load-from url #(re-frame/dispatch [follow-up-event (-> % extensions/read-extension extensions/parse)]))))
+ :extensions/load
+ (fn [{:keys [extensions follow-up]}]
+   (doseq [{:keys [url active?]} extensions]
+     (extensions/load-from url #(re-frame/dispatch [follow-up (-> % extensions/read-extension extensions/parse) active?])))))
 
 (handlers/register-handler-fx
- :extension/stage
+ :extensions/stage
  (fn [{:keys [db] :as cofx} [_ extension-data]]
    (fx/merge cofx
              {:db (assoc db :staged-extension extension-data)}
              (navigation/navigate-to-cofx :show-extension nil))))
 
 (handlers/register-handler-fx
- :extension/add
- (fn [cofx [_ data active?]]
-   (extensions/add cofx data active?)))
\ No newline at end of file
+ :extensions/add
+ (fn [cofx [_ {:keys [data]} active?]]
+   (extensions/add cofx data active?)))
+
+(handlers/register-handler-fx
+ :extensions/deactivate-all
+ (fn [cofx [_ extensions]]
+   (apply fx/merge cofx (map (fn [{:keys [name]}]
+                               (partial registry/deactivate name))
+                             extensions))))
```

Comment:
Great work, in incorporating the feedback, just one small last thing, we don't need event `:extensions/deactival-all` at all -> instead of making it event, please make it a named model function in `extensions.core` ns and in the single place where it's dispatched (function `update-extensions-state` in accounts), call the function to produce map of effects instead of dispatching event.
If the above can't be done due to circular dependencies, then ignore it for now and I will refactor ns structure and take care of that later.